app/models/vm_or_template.rb

```diff
@@ -30,6 +30,9 @@ class VmOrTemplate < ApplicationRecord
   include TenancyMixin
 
   include AvailabilityMixin
+  include SupportsFeatureMixin
+
+  supports_not :retirement, ""
```

Comment:
@jameswnl we can remove the `, ""` now. The SupportsFeatureMixin was just updated